nixos/modules/services/misc/paperless-ng.nix

```diff
@@ -30,7 +36,7 @@ let
       "-/etc/hosts"
       "-/etc/localtime"
       "-/run/postgresql"
-    ];
+    ] ++ (optional (!hasCustomRedis) config.services.redis.servers.paperless.unixSocket);
```

Comment:
The test fails for me with this line (`nix-build -A paperless-ng.tests`).
Changing the added path to `"-/run/redis-paperless"` made it work for me.

Reply:
Thanks! Fixed by adding `after = [ "redis-paperless-ng.service" ];`, which I think is a cleaner fix since the server could still fail if it doesn't find the redis socket when starting